### PR TITLE
added graph command and minor ux changes

### LINKED ISSUE #13: add command to support generating graphs
The module dependencies can be visualised with the help of a directed graph which can be generated using [Graphviz's](https://www.graphviz.org) dot command. 

We would need to generate a `.dot` file which which has all edges of the graph in a format understood by Graphviz.
This [README](https://github.com/go-modules-by-example/index/blob/master/014_mod_graph/README.md) explains the process in detail.

### LINKED ISSUE #19: sort list of dependencies
List of dependencies is printed when using `depstat list` and `depstat stats --verbose`. It'll be nice to have the list in sorted order.

### PR DESCRIPTION
**Pull request by RinkiyaKeDad**
Fixes #13, fixes #19 

- Adds a `graph` command which can be used to generate a .dot file which is compatible with Graphviz's dot command.
- Printing dependencies in sorted order.
- Remove default flags generated by cobra and config file flag
- Minor updates to command descriptions

### CODE REVIEW

File: cmd/root.go [hunk 1]
```diff
@@ -31,43 +25,3 @@ func Execute() {
 		os.Exit(1)
 	}
 }
-
-func init() {
```
Comment (RinkiyaKeDad) [inline cmd/root.go:35]: @alenkacz removing this function won't cause any problems, right? The binary is still working when I tested locally.

File: cmd/graph.go [hunk 2]
```diff
@@ -0,0 +1,42 @@
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/spf13/cobra"
+)
+
+// analyzeDepsCmd represents the analyzeDeps command
```
Comment (alenkacz) [inline cmd/graph.go]: this should be changed

File: cmd/graph.go [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+	RunE: func(cmd *cobra.Command, args []string) error {
+		depGraph, deps, _ := getDepInfo()
+		fileContents := "digraph {\ngraph [rankdir=TB, overlap=false];\n"
+		for _, dep := range deps {
+			_, ok := depGraph[dep]
+			if !ok {
+				continue
+			}
+			for _, neighbour := range depGraph[dep] {
+				fileContents += ("\"" + dep + "\" -> \"" + neighbour + "\"\n")
```
Comment (alenkacz) [inline cmd/graph.go]: what do you think about using fmt.Sprintf here rather than string concatenation?

**Review by alenkacz — APPROVED** [review]
LGTM, two small nits

**Comment by RinkiyaKeDad** [comment]
Made the changes! Thanks for reviewing @alenkacz :)

**Pull request merged**